### PR TITLE
Portfolio-Responsive-Movile-Version

### PR DESCRIPTION
**Pull request by SaveryIV**
In this project, I end:

   -The data Grid with the cards.
   -The About Myself, with the lists.
   -The buttons hover, pressed, and disconnected properties.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -314,9 +293,6 @@
   .list-work {
     width: 80%;
     border-right-width: 20%;
-  }
```
Comment (Owusu-Desmond) [inline style.css:317]: - [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But the dot divider is not present in all your project card so please make sure that you fix it :pray: 
  
  | Your Design |Figma Design |
  | :---:   | :---: | 
  | image | image |
  

- [ ] Also as you can see, all your project title's colors and and the company (for instance the "Canopy") color does not match the Figma design so make sure that you fix them

File: style.css [hunk 2]
```diff
@@ -196,7 +187,7 @@
     font-size: 15px;
     line-height: 24px;
     padding-left: 25px;
-    margin-bottom: 30pxz;
+    margin-bottom: 30;
     margin-top: 5px;
   }
```
Comment (Owusu-Desmond) [inline style.css:193]: - [ ] Please this card image it not showing so make sure that you fix it to make your work match the Figma design. :pray: 
  
  image

File: README.md [hunk 3]
```diff
@@ -1,33 +1,8 @@
...
-
-After you're finished please remove all the comments and instructions!
--->
-
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <h3><b>Portfolio Project</b></h3>
+  <h3><b>Portfolio Project Final Movile Version</b></h3>
 
 </div>
```
Comment (Owusu-Desmond) [inline README.md:8]: - [ ] Why dont you remove these comments below you have in your readme from your readme file to make it more professional :blush: 
    - "_Describe the tech stack and include only the relevant sections that apply to your project._"
    - "_Describe between 1-3 key features of the application._"

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @SaveryIV , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Nice UI design ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SaveryIV** [comment]
I Desmond, i dont now how change the color of text "CANOPY".
The other thing is i dont now why the image is like that, because in my browser seems right

**Comment by SaveryIV** [comment]
Let me pass pls because is almost exactly the same. 
This is how seems to me
image

**Comment by Owusu-Desmond** [comment]
If you are sure that the image is showing your side then it can be running
problems at my side. Please fix the  requested changes left and submit for
review again

**Comment by SaveryIV** [comment]
Can show me how pls? I tried but i can't @Owusu-Desmond

**Comment by Owusu-Desmond** [comment]
Hi @SaveryIV,

Please which of the suggested changes do
you not get it so that I can elaborate further

**Comment by SaveryIV** [comment]
> Hi @SaveryIV,
> 
> Please which of the suggested changes do you not get so that I can elaborate further

The dots between the words

**Comment by SaveryIV** [comment]
@Owusu-Desmond Okey i think i fnish now, i made the changes

**Review by Trast00 — COMMENTED** [review]
Hi @SaveryIV , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that failed linters, consider fixing the linter and making sure all of them are green, and then ask for a review. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@Trast00) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by SaveryIV** [comment]
> Hi @SaveryIV ,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that failed linters, consider fixing the linter and making sure all of them are green, and then ask for a review.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me(@Trast00) in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Now i fixed, the linters are working well

**Review by Trast00 — APPROVED** [review]
# Approved 🟢
Hi @SaveryIV , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
allfather-approves-approved

## Highlights
- Project look awsome 🍏 🌷 
- Code is clean and well organized 🧼 💯 

## Optional suggestions

- [ ] Kindly, replace the image URL at line [194](https://github.com/SaveryIV/Portfolio-Responsive/blob/master/index.html#L164) from `/resources/work-section/masks/Snapshoot PortfolioMulti-Post2-mask.png` by  `./resources/work-section/masks/Snapshoot PortfolioMulti-Post2-mask.png`. The dot at the beginning oof the URL make sure the image always display properly. This will make your project way more prettier which will improve the user experience 📱  😄 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@Trast00) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bantealem — COMMENTED** [review]
Hi @SaveryIV, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

## Highlights
- [x] No lInters error
- [x] Correct Github flow
- [x] Descriptive PR title and  Summary
- [x] Professional README
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**